resources/ory/charts/hydra/charts/hydra-maester/templates/deployment.yaml

```diff
@@ -29,6 +29,8 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
     spec:
+      seccompProfile:
```

Comment:
I think this is not valid here ([reference](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)) and should be moved to `spec.securityContext`.